onadata/apps/logger/models/xform.py

```diff
@@ -890,6 +889,11 @@ def soft_delete(self, user=None):
         self.id_string += deletion_suffix
```

Comment:
@ukanga @lincmba basically I am asking is there a reason why we do not want to rewrite lines `889` and `890` as something like:

```python
self.id_string = self.id_string[:self.MAX_ID_LENGTH - len(deletion_suffix)] + deletion_suffix
self.sms_id_string = self.sms_id_string[:self.MAX_ID_LENGTH - len(deletion_suffix)] + deletion_suffix
```

Reply:
Sounds sane, the question I have, is having the `deletion_suffix` in full important? We do have the `deleted_at` field that has the timestamp of the deletion event.